package/com.unity.formats.usd/Tests/USD.NET.Unity/InstancingTests.cs

```diff
@@ -1,7 +1,11 @@
 using System;
+using System.Collections.Generic;
+using System.IO;
```

Comment:
I don't think these are necessary anymore with the latest refactor of the test code :)